plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -169,6 +169,23 @@ func findNodesThatFit(pod *api.Pod, nodeNameToInfo map[string]*schedulercache.No
 			if err != nil {
 				return api.NodeList{}, FailedPredicateMap{}, err
 			}
+			//add the extender failed info to failedPredicateMap
+			for _, filteredNode := range filtered {
```

Comment:
This is already running in generic scheduler for every extenders! It should just require the interface to return it.
An extender can return nil or empty if unnecessary.